index.html

```diff
@@ -4,484 +4,528 @@
-              </li>
-              <li class="nav-menu__list-item list-item">
-                <a class="list-link" href="#">Service</a>
-              </li>
-              <li class="nav-menu__list-item list-item">
-                <a class="list-link" href="#">Jobs</a>
-              </li>
-            </ul>
-
-            <!-- Navigation button -->
-            <div class="hero-section__button">
-              <a class="buttons buttons__nav-button" href="#">
-                Post a vacancy
-              </a>
-            </div>
-          </nav>
-          <!-- Hero section wrapper -->
-          <div class="hero-section__wrapper">
-            <!-- Left wrapper -->
-            <div class="left-wrapper">
-              <h1 class="left-wrapper__title title">
-                Find & Apply For Your Dream Job
-              </h1>
-              <p class="left-wrapper__subtitle">
-                The website is chockablock with useful features, allowing you to
-                efficiently search and apply for roles across all sectors,
-                experience levels and locations.
-              </p>
+      <!-- Hamburger -->
+      <button class="hamburger">
+        <span class="hamburger__item"></span>
+        <span class="hamburger__item"></span>
+        <span class="hamburger__item"></span>
+      </button>
 
-              <!-- Search container -->
-              <div class="search-container">
-                <!-- Search form -->
-                <form class="search-container__search-form" method="get">
-                  <img
-                    class="search-container__search-icon"
-                    src="./src/icons/search-icon.svg"
-                    alt="search"
-                  />
-                  <input
-                    class="search-input"
-                    type="text"
-                    placeholder="Job title or Keyword"
-                    name="text"
-                    value=""
-                  />
-                </form>
+      <div class="container header-wrapper">
+        <a class="logo link-black" href="#">
+          J<span class="decor_orange">O</span>BPLY
+        </a>
 
-                <!-- Location form -->
-                <img src="./src/icons/line.svg" alt="icon" />
-                <form class="search-container__location-form" method="get">
-                  <img
-                    class="search-container__location-icon"
-                    src="./src/icons/location-icon.svg"
-                    alt="location"
-                  />
-                  <input
-                    class="location-input"
-                    type="text"
-                    placeholder="Location"
-                    name="text"
-                    value=""
-                  />
-                </form>
+        <!-- Navigation menu -->
+        <nav class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="nav-menu__list-item"><a href="#home">Home</a></li>
```

Comment:
```suggestion
            <li class="nav-menu__list-item"><a href="#">Home</a></li>
```
Remove anchors here and below